keyboards/3.0/egy-Egyp-t-k0-qwerty.xml

```diff
@@ -0,0 +1,6596 @@
+    <import base="cldr" path="47/keys-Zyyy-currency.xml"/>
+    <key id="Ayin" output="\m{Ayin}"/> 
+    <key id="ayin" output="\m{ayin}"/> 
+    <key id="Alef" output="\m{Alef}"/>
+    <key id="alef" output="\m{alef}"/>
+    <key id="Yod" output="Ꞽ"/>
+    <key id="yod" output="ꞽ"/>
+    <key id="Dunderbar" output="Ḏ"/>
+    <key id="dunderbar" output="ḏ"/>
+    <key id="AltH" output="\m{AltH}"/>
+    <key id="Alth" output="\m{Alth}"/>
+    <key id="Kunderdot" output="Ḳ"/>
+    <key id="kunderdot" output="ḳ"/>
+    <key id="Scaron" output="Š"/>
+    <key id="scaron" output="š"/>
+    <key id="Tunderbar" output="Ṯ"/>
+    <key id="tunderbar" output="ṯ"/>
+    <key id="middledot" output="·"/>
+    <key id="convert" output="\m{C}"/> <!-- Convert into Hieroglyphs -->
+    <key id="reconvert" output="\m{R}"/> <!-- Convert from Hieroglyph to Gardiner -->
+    <key id="phconvert" output="\m{P}"/> <!-- Convert from Hieroglyph to Phonetic -->
+    <key id="prevh" output="\m{Previous}"/> <!-- Convert to previous heiroglyph -->
+    <key id="nexth" output="\m{Next}"/> <!-- Convert to next heiroglyph -->
+    <key id="prevg" output="\m{PrvG}"/> <!-- Jump to previous group -->
+    <key id="nextg" output="\m{NxtG}"/> <!-- Jump to next group -->
+    <key id="close" output="\m{close}"/>
+    <key id="open" output="\m{open}"/>
+    <key id="tcub" output="\m{tcub}"/> 
+    <key id="tcue" output="\m{tcue}"/>
+    <key id="tcab" output="\m{tcab}"/>
+    <key id="tcae" output="\m{tcae}"/>
+    <key id="tcrb" output="\m{tcrb}"/>
+    <key id="tcre" output="\m{tcre}"/>
+  </keys>
+
+  <layers formId="us">
+    <layer modifiers="none">
+      <row keys="grave 1 2 3 4 5 6 7 8 9 0 hyphen equal"/>
+      <row keys="q w e r t y u i o p open-square close-square backslash"/>
+      <row keys="a s d f g h j k l semi-colon apos"/>
+      <row keys="z x c v b n m comma period slash"/>
+      <row keys="convert"/>
+    </layer>
+    <layer modifiers="shift">
+      <row keys="tilde bang at hash dollar percent caret amp asterisk open-paren close-paren underscore plus"/>
+      <row keys="Q W E R T Y U I O P open-curly close-curly pipe"/>
+      <row keys="A S D F G H J K L colon double-quote"/>
+      <row keys="Z X C V B N M open-angle close-angle question"/>
+      <row keys="space"/>
+    </layer>
+    <layer modifiers="caps">
+      <row keys="grave 1 2 3 4 5 6 7 8 9 0 hyphen equal"/>
+      <row keys="Q W E R T Y U I O P open-square close-square backslash"/>
+      <row keys="A S D F G H J K L semi-colon apos"/>
+      <row keys="Z X C V B N M comma period slash"/>
+      <row keys="convert"/>
+    </layer>
+    <layer modifiers="altR">
+      <row keys="grave 1 2 3 4 5 6 7 8 open close prevh nexth"/>
+      <row keys="Q W E R tunderbar Y U yod O P tcub tcue backslash"/>
+      <row keys="alef scaron dunderbar F G Alth J kunderdot L semi-colon ayin"/>
+      <row keys="Z X C V B N M tcab tcae slash"/>
+      <row keys="reconvert"/>
+    </layer>
```

Comment:
I would have expected unused RAlt combinations to be gaps rather than emitting, I think, the Caps layer equivalent?

Reply:
Converting to gaps